packages/data-stores/src/user/constants.ts

```diff
@@ -0,0 +1 @@
+export const STORE_KEY = 'automattic/user';
```

Comment:
Trying to decide if this store should be called `user` or `auth` 🤔

Reply:
I couldn't decide, either!

Reply:
Leaving it as `user` for the time being, we can always change it later :)